libnavigation-base/src/main/java/com/mapbox/navigation/base/utils/DecodeUtils.kt

```diff
@@ -122,6 +122,20 @@ object DecodeUtils {
         return stepsGeometryDecodeCache.getOrDecode(legStep.geometry(), precision)
     }
 
+    /**
+     * Clears the caches that were filled by invoking
+     * [stepGeometryToPoints], [completeGeometryToPoints], etc.
+     */
+    @JvmStatic
+    fun clearCache() {
```

Comment:
Why do we need to expose it? In which cases user may need to manually clear geometry caches?